build_rules.gradle

```diff
@@ -560,6 +560,10 @@ ext.applyJavaNature = {
     tasks.withType(JavaCompile) {
       options.compilerArgs += "-XepDisableWarningsInGeneratedCode"
     }
+    dependencies {
```

Comment:
Should this be outside the `enableErrorProne` block? My understanding is that these annotations are for Findbugs.

Reply:
Well these imports were mostly added to remove warnings on module-info.java files reported by error-prone. I can relocate it if you prefer but I have the impression that this scope is tighter.

Reply:
I don't have a strong preference. Maybe add a comment explaining why it's here since it's not totally obvious why ErrorProne usage should require a FindBugs dependency.

Reply:
ok